incubator/patroni/values.yaml

```diff
@@ -35,7 +35,14 @@ Etcd:
 
 persistentVolume:
   size: 1G
-  storageClass: ""
+  ## database data Persistent Volume Storage Class
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  ##
+  # storageClass: "-"
```

Comment:
Thanks @mgoodness for the update, I did not comment out the storage class is because I want to keep consistent with old chart, the old chart is using `storageClass: ""`, so here I want to use `storageClass: "-"` instead, why want to comment out the `storaggeClass` here?